electron/main.ts

```diff
@@ -354,11 +354,10 @@ ipcMain.on('callTool', async (event, { tool, wine, prefix, exe }: Tools) => {
     winePrefix = `'${protonPrefix}/pfx'`
   }
 
-  let command = `WINE=${wineBin} WINEPREFIX=${winePrefix} 
-    ${tool === 'winecfg' ? `${wineBin} ${tool}` : tool}`
+  let command = `WINE=${wineBin} WINEPREFIX='${winePrefix}' ${tool === 'winecfg' ? `${wineBin} ${tool}` : tool}`
```

Comment:
What exactly is changing here?

Reply:
I think one of the causes for the problem was that the line was spilt into two, but I am not sure about that. The main change is from "WINEPREFIX=${winePrefix}" to "WINEPREFIX='${winePrefix}'".

Reply:
To fix this kind of error, is good for you to:
1. Simulate the issue exactly as described
2. Add the fix
3. Repeat and check if the error is gone
4. Do the same but using different prefixes, wine and proton configurations.
5. Also, test with prefixes and wine paths that contains spaces on it.

If everything works fine after that, then the error might be gone and nothing was broken.

Because I played for several times with this before, and its hard to not break anything. Some people might have config from months before and we changed something that might have broken it. So updating their config files, should fix the issue most of the time.

Reply:
Thanks for the reply, I followed your steps as far as I could right now (I am on mac right now, so I couldn't check if it works with proton) and I actually found a problem with spaces in different kinds of paths. I added the fix to the pull request.